### PR TITLE
Integrate ACE and RHOSAK TCs with new feature to remove disabled cards from dashboard

### PR DESCRIPTION
**Pull request by bdattoma**
The following PR wants to modify the Anaconda and RHOSAK TCs to include the changes implemented in  [RHODS-783](https://issues.redhat.com/browse/RHODS-783). So,after the config maps are deleted, the TC will wait until the label "disabled" appears on the card and after that it will delete the card from the Dashboard.

Signed-off-by: bdattoma <bdattoma@redhat.com>

### CODE REVIEW

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
Comment (lugi0) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:85]: Are there multiple buttons with text "here"? I would try to find a different way to click them if so, the index of the correct button is prone to changing

Comment (lugi0) [inline tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:85]: Are there multiple buttons with text "here"? I would try to find a different way to click them if so, the index of the correct button is prone to changing

File: tests/Resources/Page/HybridCloudConsole/Rhosak.robot
Comment (lugi0) [inline tests/Resources/Page/HybridCloudConsole/Rhosak.robot:7]: Ordering of Resource/Library

**Review by lugi0 — COMMENTED** [review]
Left a couple comments, lgtm otherwise

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
Comment (bdattoma) [reply tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:85]: unfortunately there are two button with text "here" one after the other. I didn't find any attribute to use to capture the link differently. However, it probably gonna change because of a file issue which may change the UI. So, for now i would keep it as it is and after the changes are released I will change the logic here.

File: tests/Resources/Page/HybridCloudConsole/Rhosak.robot
Comment (bdattoma) [reply tests/Resources/Page/HybridCloudConsole/Rhosak.robot]: done

**Review by lugi0 — DISMISSED** [review]
lgtm

File: tests/Resources/Page/HybridCloudConsole/Rhosak.robot
Comment (tarukumar) [inline tests/Resources/Page/HybridCloudConsole/Rhosak.robot:197]: In line 194- you are going back to RHODS windows.
In line 195 - you are closing the browser
In line  196~197 -> This keyword open new browser and does login

I don't understand why you have to close and relaunch again? am i missing anything?

After Switch window can't we use keyword in 198 . it could relaunching of dashboard

File: tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
Comment (bdattoma) [reply tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot:13]: test variables file reports "ocp" too. I don't know what's best here. Maybe when we'll define a finale policy for variables name for rhods and odh we could change it accordingly

File: tests/Resources/Page/HybridCloudConsole/Rhosak.robot
Comment (bdattoma) [reply tests/Resources/Page/HybridCloudConsole/Rhosak.robot:197]: I know it sounds meaningless, but it helps in make the TC quicker because of the Dashboard refresh mechanism.We spare up to 1 min I think

Comment (tarukumar) [reply tests/Resources/Page/HybridCloudConsole/Rhosak.robot:197]: Hmmm..if we are closing the browser, i don't think we have to Switch to RHODS dashboard(line 194)

Comment (bdattoma) [reply tests/Resources/Page/HybridCloudConsole/Rhosak.robot:197]: hmm that's right, it was there from prev version of the TC. I can remove it

Comment (bdattoma) [reply tests/Resources/Page/HybridCloudConsole/Rhosak.robot:196]: done

**Review by lugi0 — DISMISSED** [review]
lgtm

**Comment by bdattoma** [comment]
! do not merge before #190

File: tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot
Comment (bdattoma) [reply tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot:86]: @jgarciao I could try to put together some lines into single keywords. But I would postpone this effort in a different PR

Comment (jgarciao) [reply tests/Tests/600__ai_apps/650__rhosak/650__rhosak.robot:86]: ok!

**Review by jgarciao — APPROVED** [review]
LGTM (pending refactoring of long test in another PR)

**Comment by bdattoma** [comment]
Anaconda CE PR Testing: [JenkinsJob#406](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/406/) _Success_

RHOSAK PR Testing: _coming soon_

**Review by milindw96 — APPROVED** [review]

**Review by jgarciao — APPROVED** [review]

**Pull request merged**
